Assignments/stu5/assignments.py

```diff
@@ -112,6 +117,23 @@ def replace_period(sentence):
     retval = sentence.replace (".", "!")
     return retval
 
+def slice_it(array):
+    retval=""
+    for i in array:
+        retval += str(i[:2])
+    print(retval)
```

Comment:
We should return `retval` here, not return it.

Reply:
(not print it)

Reply:
PR UPDATED